.github/workflows/publish-generator.yml

```diff
@@ -235,3 +241,65 @@ jobs:
           platforms: linux/amd64,linux/arm64
           push: true
           tags: fernapi/fern-postman:${{env.VERSION}}
+
+  ruby-model:
+    runs-on: ubuntu-latest
+    needs: versions-changed
+    if: ${{ needs.changes.outputs.ruby-model == 'true'}}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: "yarn"
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: fernapi
+          password: ${{ secrets.FERN_API_DOCKERHUB_PASSWORD }}
+
+      - name: Put Version in Env
+        run: |
+          projectVersion=$(cat generators/ruby/sdk/VERSION)
+          echo $projectVersion
+          echo "VERSION=$projectVersion" >> $GITHUB_ENV
+
+      - name: Build and push Ruby Model Docker image
+        run: |
+          projectVersion=${{ env.VERSION }}
+          yarn workspace @fern-api/fern-ruby-model dockerTagVersion "$projectVersion"
+          docker push fernapi/fern-ruby-model:"$projectVersion"
+
+  ruby-sdk:
+    runs-on: ubuntu-latest
+    needs: versions-changed
+    if: ${{ needs.changes.outputs.ruby-model == 'true'}}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: "yarn"
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: fernapi
+          password: ${{ secrets.FERN_API_DOCKERHUB_PASSWORD }}
+
+      - name: Put Version in Env
+        run: |
+          projectVersion=$(cat generators/ruby/sdk/VERSION)
+          echo $projectVersion
+          echo "VERSION=$projectVersion" >> $GITHUB_ENV
+
+      - name: Build and push Ruby SDK Docker image
+        run: |
+          projectVersion=${{ env.VERSION }}
+          yarn workspace @fern-api/fern-ruby-sdk dockerTagVersion "$projectVersion"
+          docker push fernapi/fern-ruby-sdk:"$projectVersion"
```

Comment:
can we publish multiplatform dockers?

Reply:
good call! I didn't see we were doing that!